multisig/simulations/arbitrumone/vip-013/index.ts

```diff
@@ -0,0 +1,451 @@
+        poolRegistry = await ethers.getContractAt(POOL_REGISTRY_ABI, POOL_REGISTRY);
+        registeredPools = await poolRegistry.getAllPools();
+      });
+
+      it("should have 2 pools(Core and Liquid staked ETH)", async () => {
+        expect(registeredPools).to.have.lengthOf(2);
+      });
+
+      it("should register Liquid staked ETH pool in PoolRegistry", async () => {
+        const pool = registeredPools[1];
+        expect(pool.name).to.equal("Liquid Staked ETH");
+        expect(pool.creator).to.equal(GUARDIAN);
+        expect(pool.comptroller).to.equal(COMPTROLLER_LIQUID_STAKED_ETH);
+      });
+
+      it("should register vTokens in Liquid staked ETH pool Comptroller", async () => {
+        const comptroller = await ethers.getContractAt(COMPTROLLER_ABI, COMPTROLLER_LIQUID_STAKED_ETH);
+        const poolVTokens = await comptroller.getAllMarkets();
+        expect(poolVTokens).to.have.lengthOf(3);
+        expect(poolVTokens).to.include(vTokens.vwstETH_Liquid_staked_ETH);
+        expect(poolVTokens).to.include(vTokens.vweETH_Liquid_staked_ETH);
+        expect(poolVTokens).to.include(vTokens.vWETH_Liquid_staked_ETH);
+      });
+
+      for (const [symbol, { underlying }] of Object.entries(vTokenState) as [VTokenSymbol, VTokenState][]) {
+        it(`should register ${symbol} in PoolRegistry`, async () => {
+          const registeredVToken = await poolRegistry.getVTokenForAsset(vTokenState[symbol].comptroller, underlying);
+          expect(registeredVToken).to.equal(vTokens[symbol]);
+        });
+      }
+    });
+
+    describe("Ownership", () => {
+      for (const [symbol, address] of Object.entries(vTokens) as [VTokenSymbol, string][]) {
+        it(`should transfer ownership of ${symbol} to GUARDIAN`, async () => {
+          const vToken = await ethers.getContractAt(VTOKEN_ABI, address);
+          expect(await vToken.owner()).to.equal(GUARDIAN);
+        });
+      }
+    });
+
+    describe("ProtocolShareReserve", () => {
+      for (const [symbol, address] of Object.entries(vTokens) as [VTokenSymbol, string][]) {
+        it(`should set PSR for ${symbol}`, async () => {
+          const vToken = await ethers.getContractAt(VTOKEN_ABI, address);
+          expect(await vToken.protocolShareReserve()).to.equal(PSR);
+        });
+      }
+    });
+
+    describe("Initial supply", () => {
+      for (const [symbol, params] of Object.entries(riskParameters) as [VTokenSymbol, RiskParameters][]) {
+        it(`should mint initial supply of ${symbol} to ${params.vTokenReceiver}`, async () => {
+          // Since we're distributing 1:1, decimals should be accounted for in the exchange rate
+          const expectedSupply = parseUnits(params.initialSupply, 8);
+          const vToken = await ethers.getContractAt(VTOKEN_ABI, vTokens[symbol]);
+          expect(await vToken.balanceOf(params.vTokenReceiver)).to.equal(expectedSupply);
+        });
+      }
+    });
+
+    describe("Risk parameters", () => {
```

Comment:
WDYT if we moe this to `checks/`? It seems useful for other VIPs

I did it [here](https://github.com/VenusProtocol/vips/blob/486096995ab28477cb54c69ec0e95cbc2b3f7e85/src/vip-framework/checks/checkRiskParameters.ts), so I think no changes are needed for this PR. Just want to discuss how we want to do risk parameter checks in future.

Reply:
Yes, we should keep the generic checks at one place so that no need to change them in every VIP.
For future we should use them as per your reference to maintain the consistency.